demo/voice_cmd.dic

```diff
@@ -1,11 +1,9 @@
 BACK	B AE K
-BACKWARD	B AE K W ER D
 FORWARD	F AO R W ER D
 FULL	F UH L
 HALF	HH AE F
-HALT	HH AO L T
```

Comment:
Did you remove these two so as to avoid confusion between "BACK" <-> "BACKWARD" and "HALF" <-> "HALT"?